lib/ansible/modules/net_tools/basics/get_url.py

```diff
@@ -49,7 +49,8 @@
   tmp_dest:
     description:
       - Absolute path of where temporary file is downloaded to.
-      - Defaults to C(TMPDIR), C(TEMP) or C(TMP) env variables or a platform specific value.
+      - Since 2.5 the path defaults to the module's temp dir or C(TMPDIR), C(TEMP) or C(TMP) env variables or a platform specific value.
+      - Before 2.5 the module's temp dir was not part of the default.
```

Comment:
The only thing left to fix in this PR is the documentation here.  I think that the way it's currently coded, in 2.5 the path default's to ansible's remote_tmp path.  In 2.5 or earlier it defaults to the platform setting.

Reply:
isn't that exactly what it is saying?

Reply:
It currently says that in 2.5, it can be the module's temp dir or a platform setting (via one of hte environment variables).  But it looks like the code will only use the remote_tmp as the default with no code path that would reach the platform settings.

Reply:
Something like:
```
- When run on Ansible 2.5 or greater, path defaults to ansible's remote_tmp setting
- When run on Ansible prior to 2.5, it defaults to C(TMPDIR), C(TEMP) or C(TMP) env variables or a platform specific value.
```